04.ls/ls_command.rb

```diff
@@ -5,47 +5,52 @@
 COLUMN_COUNT = 3
 
 def main
-  files = if a_option?
-            fetch_files(option: 'a')
+  options = load_options
+  files = fetch_files
+
+  files = if options.key?(:a)
+            files
+          else
+            files.filter { |file| file unless file.start_with?('.') }
```

Comment:
```suggestion
            files.filter { |file| !file.start_with?('.') }
```

ブロックは真偽値を返すだけでいいので、上のようなコードでいいはずです。
あと、filterの代わりにrejectを使うと`!`も無くせます。